configure.ac

```diff
@@ -724,22 +724,42 @@ if test $ac_lib_slang = auto; then
 		[ac_lib_slang=no],
 		[]
 	)
```

Comment:
I think you meant to remove this `AC_CHECK_LIB` call like you have done for ncurses below? It's useless anyhow because the following `PKG_CHECK_MODULES` call overwrites the result.

Reply:
Yep, just forgot to remove it after I created the `PKG_CHECK_MODULES`.  Fixed.